helm-charts/easegress/README.md

```diff
@@ -58,3 +55,18 @@ helm uninstall easegress -n easegress
 kubectl delete pvc easegress-pv-easegress-0 -n easegress
 # same for easegress-pv-easegress-i...n
 ```
+
+## Parameters
+
+The following table lists the configurable parameters of the MegaEase Easegress.
+
+| Key | Type | Default | Description |
```

Comment:
This table duplicates the information in `values.yaml` file, forcing to change two places, when something changes here. Given that [values.yaml](https://helm.sh/docs/chart_best_practices/values/) is common to all Helm installations, I would consider removing this table and only referencing values.yaml here, like `Helm parameters and default values are listed in values.yaml`.

Reply:
in kubesphere, https://github.com/kubesphere/helm-charts/tree/master/src/ every readme has parameter part to show what the parameter are...

Reply:
So i will keep the parameters...